### DISCUSSION TITLE
Como descobrir o nome de um componente mais facilmente

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1690] [Assuntos gerais]
Boa tarde, pessoal.

Como faço para descobrir mais facilmente qual componente usa qual cor?

Exemplo, nesta tela de login o fundo do campo de usuário/senha está em um tom azul. Gostaria de mudar para outra cor.

![image](https://user-images.githubusercontent.com/46979335/233474305-ef76ffed-46f5-460e-95a9-8e8d13c492e2.png)

Existe algum macete em como descobrir qual propriedade do CSS se refere a esse fundo para que eu possa alterá-la? Exemplo --color-brand-01-base.

Desculpe a ignorância, sou iniciante em POUI.

Grato.

**Comment by jcorrea97** [comment]
Boa tarde @gtrabach, no nosso contrutor de temas deixamos alguns exemplos de como customizar seu componente, inclusive o input:
https://po-ui.io/construtor-de-temas

Não deixamos todas as variavéis possiveis no construtor, mas para acessar as todas as customizações você pode inspecionar o elemento no seu navegador e ir diretamente no seletor, por exemplo o po-input:

![image](https://user-images.githubusercontent.com/57047381/233985460-c55fdd1f-d96a-4b26-9a8e-2494ee4ae2d1.png)

**Reply by gtrabach** [reply]
Perfeito, obrigado.
